catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/internal/SubscriptionFilterVisitor.java

```diff
@@ -379,6 +377,19 @@ public Object visit(During filter, Object data) {
         return returnPredicate;
     }
 
+    private DateType getDateType(String attr) {
```

Comment:
❗️ We should make this a `static` method on `DateType` so it can be used instead of `valueOf()`.